### DISCUSSION TITLE
[ios] bug

### DISCUSSION THREAD

**Discussion by Zero001235** [ppy/osu#23195] [Q&A]
went through the same map several times and the game began to enter the menu itself

**Comment by Zero001235** [comment]
[RPReplay_Final1681325058.zip](https://github.com/ppy/osu/files/11215061/RPReplay_Final1681325058.zip)

**Comment by peppy** [comment]
You probably pressed the "hold to pause: button in the bottom right of the screen.

**Reply by Zero001235** [reply]
no, I went through the map with a stylus and never pressed the menu button, during the sliders the game itself clamps this button

**Comment by Zero001235** [comment]
I removed the protective film from the tablet, but it did not help, osu keeps going to the menu even when I don't touch the screen
